Resources/public/js/views/services/plugins/ez-versionsplugin.js

```diff
@@ -0,0 +1,155 @@
+        },
+
+        /**
+         * Loads versions list for content given in event facade. When loading of versions
+         * is finished, then it is set in `versions` attribute of the event target.
+         *
+         * The version attribute should be an object with the given properties: `archived`,
+         * `published` and `draft`
+         *
+         * @method _loadVersions
+         * @private
+         * @param {EventFacade} e loadVersions event facade
+         * @param {eZ.Content} e.content the content
+         *
+         */
+        _loadVersions: function (e) {
+            var service = this.get('host'),
+                capi = service.get('capi'),
+                options = {
+                    api: capi,
+                };
+
+            e.content.loadVersions(options, Y.bind(function (error, versions) {
+                if (error) {
+                    e.target.set('loadingError', true);
+                } else {
+                    e.target.set('loadingError', false);
+                    e.target.set('versions', this._sortVersions(versions));
+                }
+            }, this));
+        },
+
+        /**
+         * Sorts an array of version info by status
+         *
+         * @method _sortVersions
+         * @protected
+         * @param versions {Array} of eZ.VersionInfo
+         * @return {Object} of sorted versions by status:
+         *              struct.<status_name>: {Array} of eZ.VersionInfo
+         */
+        _sortVersions: function(versions) {
+            var versionsByStatus = {};
+
+            versions.forEach(function (version) {
+                if ( !versionsByStatus[version.get('status')]) {
+                    versionsByStatus[version.get('status')] = [];
+                }
+                versionsByStatus[version.get('status')].push(version);
+             });
+
+            return versionsByStatus;
+        },
+
+        /**
+         * Creates a draft based on an archived version and redirect to the edit page in order
+         * to edit it.
+         *
+         * @method _createDraftFromArchivedVersion
+         * @private
+         * @param {EventFacade} e
+         * @param {eZ.Content} e.content the content
+         * @param {String} e.versionNo of the archived version. Ex: 42
+         *
+         */
+        _createDraftFromArchivedVersion: function (e) {
+            var options = {
+                    api: this.get('host').get('capi'),
+                },
+                app = this.get('host').get('app'),
+                content = e.content;
+
+            content.createDraft(options, e.versionNo, Y.bind(function (error, response) {
+                var routeName = 'editContentVersion',
+                    routeParams = {
+                        id: content.get('id'),
+                        languageCode: response.document.Version.VersionInfo.initialLanguageCode,
```

Comment:
related to https://github.com/ezsystems/PlatformUIBundle/pull/611/files#r67123398, then here you could just write `version.get('initialLanguageCode');`